inst/extdata/docuData/singlecellmultimodalv4.csv

```diff
@@ -0,0 +1,6 @@
+"DataProvider","TaxonomyId","Species","SourceUrl","SourceType","SourceVersion","DataType","Maintainer"
+"Dept. of Bioinformatics, The Babraham Institute, United Kingdom","10090","Mus musculus","https://cloudstor.aarnet.edu.au/plus/s/Xzf5vCgAEUVgbfQ","RDS","1.0.0","mouse_gastrulation","Marcel Ramos <marcel.ramos@roswellpark.org>"
+"Dept. of Bioinformatics, The Babraham Institute, United Kingdom","10090","Mus musculus","https://cloudstor.aarnet.edu.au/plus/s/Xzf5vCgAEUVgbfQ","RDS","2.0.0","mouse_gastrulation","Marcel Ramos <marcel.ramos@roswellpark.org>"
+"Dept. of Molecular Genetics, Allen Institute for Brain Science, United States","10090","Mus musculus","https://www.dropbox.com/sh/avj4nrd4la5i88u/AACafWwBbE-xsLvOGDwRZDpYa?dl=0","TEXT","1.0.0","mouse_visual_cortex","Dario Righelli <dario.righelli@gmail.com>"
```

Comment:
Is this supposed to be 'TEXT' or 'TXT' ? Did you run 'ExperimentHubData::makeExperimentHubMetadata'? Please use the functionality I've provided and it will help you run that function after the metadata is generated.